### DISCUSSION TITLE
Send dialog box Size

### DISCUSSION THREAD

**Discussion by MacConin** [Extended-Thunder/send-later#706] [General]
Any possibility to reduce the size of the dialog box text? See screenshot. Its ..big..
<img width="1069" alt="Bildschirmfoto 2024-04-25 um 15 51 41" src="https://github.com/Extended-Thunder/send-later/assets/662005/7a1dce0c-0f51-4a20-af8e-0c8705a0b7e2">

**Comment by jikamens** [comment]
If you shrink the box when it's open then it should remember the new size the next time it pops up.
You can also use Ctrl-i and Ctrl-+ to zoom out or in, and that is also remembered between invocations.

**Comment by MacConin** [comment]
ah, Jonathan,  ⌘- did the job  :) Many thx
